core/api/src/app/prices/stablesats-accept-quote.ts

```diff
@@ -0,0 +1,15 @@
+import { QuoteId } from "@/domain/quotes/index.types"
+import { QuotesService } from "@/services/quotes"
+
+export const acceptStableSatsQuote = async ({
+  quoteId,
+}: {
+  quoteId: QuoteId
+}): Promise<true | ApplicationError> => {
+  if (!quoteId || typeof quoteId !== "string") {
+    return new Error("Invalid quote ID provided")
+  }
```

Comment:
you need to create a checkedToQuoteId in domain also please dont use generic errors in app layer